src/App.jsx

```diff
@@ -14,33 +17,99 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+const SORT_FIELD_ALPHABET = 'alphabet';
+const SORT_FIELD_LENGTH = 'length';
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+function getPreparedGoods(goods, isLight, isReversed) {
```

Comment:
you can manage this with another `SORT_FIELD`. basically `NONE` would be good for it
```suggestion
function getPreparedGoods(goods, isReversed) {
```